egs2/TEMPLATE/enh1/enh.sh

```diff
@@ -201,6 +201,9 @@ fi
 [ -z "${valid_set}" ] &&   { log "${help_message}"; log "Error: --valid_set is required"  ; exit 2; };
 [ -z "${test_sets}" ] && { log "${help_message}"; log "Error: --test_sets is required"; exit 2; };
 
+# Extra files for enhancement process
+utt_extra_files="utt2category text utt2lang"
```

Comment:
Just a note: we need to introduce these files in the [speech enhancement documentation](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/enh1/README.md) in later PRs.

Reply:
Cool. Just updated and added the description for utt2category.